azext_edge/edge/providers/orchestration/resources/schema_registries.py

```diff
@@ -257,9 +257,10 @@ def delete(
             return
 
         with console.status("Working..."):
-            return self.ops.begin_delete(
+            poller = self.ops.begin_delete(
                 resource_group_name=resource_group_name, schema_registry_name=schema_registry_name, schema_name=name
             )
+            return poller.result()
```

Comment:
We should use `wait_for_terminal_state` to process the poller result. Look for existing examples.